src/handlers/assign/tests/tests_candidates.rs

```diff
@@ -71,15 +72,28 @@ impl AssignCtx {
         self
     }
 
+    async fn set_previous_reviewers(mut self, users: &[&User]) -> Self {
+        let mut db = self.test_ctx.db_client_mut();
+        for user in users {
```

Comment:
Let's just create a set out of `users` and store it into the DB in one go, instead of iterating over each user. The name of the function is called `set`, not add or something, so it should overwrite the DB contents.